### ISSUE TITLE
Black font in project manager

### ISSUE THREAD

**Issue by Cykyrios** [godotengine/godot#69139]
### Godot version

v4.0.beta.custom_build [093dea21e]

### System information

Linux Mint 21, Vulkan, NVidia GTX 1060

### Issue description

All text is displayed in a black font since #67593 (I bisected to this commit) instead of the usual white, see screenshots below. I have also tested on a fresh virtual machine running Linux Mint 20.2, but did not encounter this issue (note that the VM does not support Vulkan).
![image](https://user-images.githubusercontent.com/53737317/203868966-41f82fad-a7ec-4ef7-883e-669d559fb30c.png)
![image](https://user-images.githubusercontent.com/53737317/203868972-55448c83-bbdf-42e5-a41f-182f675d82e8.png)
The issue also occurs with the default theme.

### Steps to reproduce

Open the project manager on either beta4 (no issue) or beta5 (where issue started appearing).

### Minimal reproduction project

Not applicable.

**Comment by Calinou** [comment]
I can't reproduce this on 4.0.beta 49cc12bf8 (Fedora 36, `OpenGL Renderer: AMD Radeon RX 6900 XT (sienna_cichlid, LLVM 14.0.0, DRM 3.48, 6.0.8-200.fc36.x86_64)`):

![image](https://user-images.githubusercontent.com/180032/203871912-34cfd658-4fb2-4d04-9520-5fbfd602a6e1.png)

**Comment by Cykyrios** [comment]
I didn't know about self-contained mode, interesting. And what do you know, it actually fixes the issue, so I guess I'll have to have a look at the editor settings or reset them altogether.

**Comment by Calinou** [comment]
If you're using a custom editor theme, this is likely the culprit.

**Comment by Cykyrios** [comment]
Just for clarification, I only changed 2 things: base color and accent color. I do not use a theme file. Also, I went to ~/.local/share/godot but I can't find editor related files (only app_userdata, export_templates, shader_cache, as well as mono and templates from Godot 3.x), wasn't this the folder where settings get saved? (the docs say so) Is there another place I should look if I want to "reset" the editor?
![image](https://user-images.githubusercontent.com/53737317/204037745-8440528a-8695-4547-8760-7224a1765bec.png)
